suma-storage-proxy

```diff
@@ -53,21 +58,23 @@ create_partition $storage_disk
 info "Creating $storage_fs filesystem"
 create_filesystem $storage_disk $storage_fs
 
-info "Mounting storage at $storage_location"
-mount_storage $storage_disk $storage_location
+info "Mounting storage at $storage_location_tmp"
+mount_storage $storage_disk $storage_location_tmp
 
-if systemctl status squid.service >/dev/null ; then
-    squid_running=yes
-    info "Stopping squid service"
-    systemctl stop squid.service
+if systemctl status uyuni-proxy-httpd.service >/dev/null ; then
```

Comment:
This appears to be superfluous because of the change on line 38. If the `mgrpxy` executable is found the script exits.